backends/p4tools/modules/testgen/options.cpp

```diff
@@ -154,6 +153,22 @@ TestgenOptions::TestgenOptions()
         "target-specific port variable, it will overflow. Default behavior is delegated to the "
         "test back end. Some test back ends may restrict the available port ranges.");
 
+    registerOption(
+        "--skip-control-plane-entities", "skippedControlPlaneEntities",
+        [this](const char *arg) {
+            // Convert the input into a StringStream and split by comma (',').
+            std::stringstream argStream(arg);
+            while (argStream.good()) {
+                std::string substr;
+                std::getline(argStream, substr, ',');
+                skippedControlPlaneEntities.emplace(substr);
+            }
+            return true;
+        },
+        "Specify the set of control plane entities where P4Testgen should not generate a "
```

Comment:
nit: technically "where" -> "for which" and you would remove the 'for' at the end of the sentence.